ansible-scylla-node/defaults/main.yml

```diff
@@ -208,17 +208,20 @@ scylla_yaml_params:
   internode_compression: all
 
 # SSL configuration for Scylla nodes:
-# If no certificates are present will generate a selfsigned CA and node certificates locally and push them out to all the nodes.
-# WARNING: if a local ./ssl/ directory is not present and populated by existing certificates, it will be generated
-# and certificates on the nodes can ge overwritten with new certificates.
+# localhost_cert_path is the path where the certificates will be stored in the localhost before they're coppied to the nodes.
+# cert_path is the path where the certificates will be placed in the scylla nodes.
+# If no certificates are present, this playbook will generate a selfsigned CA and node certificates locally and push them out to all the nodes.
+# WARNING: if a local ${localhost_cert_path}/ssl/ directory is not present and populated by existing certificates for all nodes, it will be generated
+# and certificates on the nodes can get overwritten with new certificates.
 # To use existing certificates:
-# 1. To use your own CA, place it in ./ssl/ca/{{ scylla_cluster_name }}-ca.pem and derived per-node certificates wll
-# be generated from it, if they are not present
-# 2. To use your own node certificates, create a directory per hostname (same as in the inventory) in ./ssl/{{ hostname }}
-# and place the .pem file as {{ hostname }}.pem
-# The playbook will use the CA certificate to generate a signed host certificate for the .pem file and push them into the nodes
+# 1. To use your own CA, place it in ${localhost_cert_path}/ssl/ca/{{ scylla_cluster_name }}-ca.crt and its respective key in
+# ${localhost_cert_path}/ssl/ca/{{ scylla_cluster_name }}-ca.pem and derived per-node certificates will be generated from it, if they are not present.
+# 2. To use your own node certificates, create a directory per hostname (same as in the inventory) in ${localhost_cert_path}/ssl/{{ hostname }}
+# and place the .pem and .crt files as {{ hostname }}.pem and {{ hostname }}.crt respectively and then place the CA you used to sign the certificates in
+# ${localhost_cert_path}/ssl/ca/{{ scylla_cluster_name }}-ca.crt and its respective key in ${localhost_cert_path}/ssl/ca/{{ scylla_cluster_name }}-ca.pem.
 # To skip this configuration entirely, set enabled: false for both options or comment out the scylla_ssl variable.
 scylla_ssl:
+  localhost_cert_path: .
```

Comment:
This is just as bad as it was before.
The value of `scylla_ssl.localhost_cert_path` is going to be `.` when you would use it. However the actual directory `.` is going to resolve to would depend what is going to be the value of `$PWD` and that particular point of the execution. 

You *must* resolve this to a concrete path instead of using `.`.

Reply:
Done